ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -408,22 +409,74 @@ def serialize(self) -> JsonObject:
             except Exception as e:
                 self.display_context.add_error(e)
 
-        return {
-            "workflow_raw_data": {
-                "nodes": cast(JsonArray, nodes_dict_list),
-                "edges": edges,
-                "display_data": self.display_context.workflow_display.display_data.dict(),
-                "definition": {
-                    "name": self._workflow.__name__,
-                    "module": cast(JsonArray, self._workflow.__module__.split(".")),
-                },
-                "output_values": output_values,
+        # Serialize workflow-level trigger if present
+        trigger_data: Optional[JsonObject] = self._serialize_workflow_trigger()
+
+        workflow_raw_data: JsonObject = {
+            "nodes": cast(JsonArray, nodes_dict_list),
+            "edges": edges,
+            "display_data": self.display_context.workflow_display.display_data.dict(),
+            "definition": {
+                "name": self._workflow.__name__,
+                "module": cast(JsonArray, self._workflow.__module__.split(".")),
             },
+            "output_values": output_values,
+        }
+
+        if trigger_data is not None:
+            workflow_raw_data["trigger"] = trigger_data
```

Comment:
[blocking] our django backend is expecting this field to be plural and at the top level: https://github.com/vellum-ai/vellum/blob/4d0f6201dde3b345590fe8573e2b44eaf72c7577/django/app/domain_models/types/workflows.py#L699